README.md

```diff
@@ -39,7 +39,166 @@ or
 
 ### Configuration
 
+### Create a config.json
```

Comment:
Create a config.json and Configuration have the same header "level" in markdown.

Reply:
fixed by eaa67e080bb89941cb69aceabd46b465d045aa3c